.maintain/gitlab/check_companion_build.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env sh
+#
+# check if a pr is compatible with companion pr or master if not available
+#
+# to override one that was just mentioned mark companion pr in the body of the
+# pr like
+#
+# $REPO companion: $ORGANISATION/$REPO#567
+#
+# $ORGANISATION and $REPO are set using $1 and $2. You can also specify a custom
+# build command with $3
+# So invoke this script like:
+# ./check_companion_build.sh paritytech polkadot 'cargo test --release'
+
+set -e
+
+ORGANISATION=$1
+REPO=$2
+BUILDSTRING=${3:-cargo test --all --release}
```

Comment:
`--all` is deprecated, please use `--workspace`

Reply:
What? When was this deprecated?

Reply:
it's not turned off completely, but throws a warning when you use `--all` for quite a while.